tests/installation/sle11_ncc.pm

```diff
@@ -7,11 +7,49 @@ sub run(){
 
     assert_screen 'novell-customer-center', 30;
 
-    # configure later
-    send_key "alt-c";
+    # configure ncc now
+    if (get_var("NCC")) {
+        send_key $cmd{'next'};
```

Comment:
Please consider moving the old test into a sle11_skip_ncc.pm and have main.pm decide. This way it's clearer what the test is doing.